rtorrent segfaults with latest ncurses (5.8).
this patch fixes the issue.

https://bugs.gentoo.org/show_bug.cgi?id=358271
https://bugzilla.redhat.com/show_bug.cgi?id=682454
http://libtorrent.rakshasa.no/ticket/2518
--- src/display/canvas.h
+++ src/display/canvas.h
@@ -48,7 +48,7 @@
 public:
   typedef std::vector<Attributes> attributes_list;
 
-  Canvas(int x = 0, int y = 0, int width = 0, int height = 0);
+  Canvas(int x = 0, int y = 0, int width = 1, int height = 1);
   ~Canvas() { delwin(m_window); }
 
   void                refresh()                                               { wnoutrefresh(m_window); }
